components/edit-collective/sections/Tiers.js

```diff
@@ -34,6 +35,8 @@ const { FUND, PROJECT, EVENT } = CollectiveType;
 const { TIER, TICKET, MEMBERSHIP, SERVICE, PRODUCT, DONATION } = TierTypes;
 const { FIXED, FLEXIBLE } = AmountTypes;
 
+const CRYPTO_CONTRIBUTIONS_ENABLED = parseToBoolean(process.env.CRYPTO_CONTRIBUTIONS_ENABLED);
```

Comment:
I don't think this is needed in the end. The flag at the host level is good enough. No feature flag needed.

Reply:
@znarf : Well, if we don't include this, then the hosts might see this feature before being fully implemented. So they could try out the crypto flow and get errors. Therefore I think we should keep it so as to prevent half baked features going to production. WDYT? 🤔